OxfordBodleian/Juel-Jensen/BDLaethe36.xml

```diff
@@ -110,43 +139,113 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc>
+                                        Scribal corrections are frequent throughout the text, sometimes interlineally (e.g. <locus target="#46r #47v"/>), sometimes over erasure in black ink (e.g. <locus target="17v"/>).
+                                        Scribbles in pale or deep black ink are found on <locus target="#21v #22v #25v #26r #29v #30r"/>.
+                                        Pen trials in pale or deep black ink are found on <locus target="#1v #2r #47v #48r #51r"/>.
+                                    </desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>Groups of ten psalms are separated with a chain of red and black dots (<locus target="#8r #16r #24v #36r #45r #52r"/>).
+                                        Stichometry is indicated in red ink by the word <foreign xml:lang="gez">ቃሉ፡</foreign> followed by a numeral:
+                                        <q xml:lang="gez">ቃሉ፡ ፳፯</q> (Ps 59, <locus target="#51v"/>),
+                                        <q xml:lang="gez">ቃሉ፡ ፲፯</q> (Ps 60, <locus target="#51v"/>), <q xml:lang="gez">ቃሉ፡ ፴</q> (Ps 61, <locus target="#52v"/>), 
+                                        <q xml:lang="gez">ቃሉ፡ ፳፬</q> (Ps 62, <locus target="#53r"/>), <q xml:lang="gez">ቃሉ፡ ፳፬</q> (Ps 63, <locus target="#53v"/>), 
+                                        <q xml:lang="gez">ቃሉ፡ ፴፫</q> (Ps 64, <locus target="#54v"/>), <q xml:lang="gez">ቃሉ፡ ፴፫</q> (Ps 65, <locus target="#55v"/>), 
+                                        <q xml:lang="gez">ቃሉ፡ ፲፭</q> (Ps 66, <locus target="#56r"/>), <q xml:lang="gez">ቃሉ፡ ፹፯</q> (Ps 67, <locus target="#58r"/>),
+                                        <q xml:lang="gez">ቃሉ፡ ፸፯</q> (Ps 68, <locus target="#60r"/>), <q xml:lang="gez">ቃሉ፡ ፲፩</q> (Ps 69, <locus target="#60r"/>).
+                                    </desc> 
+                                </item>
+                                <item xml:id="e4">
+                                    <desc>Short titles of the psalms are written in small script in black ink on the upper margin of many leaves, e.g. 
+                                        <q xml:lang="gez">ዘዳዊት፡ ሶበ፡ ሰደዶ፡ አቤሴሎም፡ ወልዱ፡</q> on <locus target="#2r"/> (Ps 3) and 
+                                        <q xml:lang="gez">ዘዳዊት፡ ሶበ፡ አበሰ፡ በብእሲተ፡ ኦርዮ፡</q> on <locus target="#3v"/> (Ps 6).
+                                        Other titles are found on <locus target="#4r #5v #8r #8v #9r #10r #11r #11v #14r #16r #18r #20v #33v #44r #45v #60r"/>.
+                                    </desc> 
+                                </item>
+                                <item xml:id="e5">
+                                    <desc>A note <q xml:lang="en">"... 1973"</q> is written in pencil in the upper inner corner of <locus target="#60v"/>, possibly by <persName ref="PRS5782JuelJen"/>.
```

Comment:
It could be! ;)

Reply:
It could be - the photo is a bit blurry - again I think we should create a list of items that we think I should rephotograph (when this will be possible) adding shelfmark and folios you need?